app/components/primer/octicon_component.rb

```diff
@@ -15,9 +15,10 @@ class OcticonComponent < Primer::Component
 
     def initialize(icon:, size: SIZE_DEFAULT, **kwargs)
       @icon, @kwargs = icon, kwargs
-
+      classes = class_names(@kwargs.delete(:class), @kwargs.delete(:classes))
+      @kwargs[:class] = class_names(Primer::Classify.call(**@kwargs)[:class], classes)
+      @kwargs = @kwargs.except(*Primer::Classify::VALID_KEYS)
```

Comment:
Is there a specific case this line this is meant to filter out?

Reply:
@joelhawksley if you don't remove keys like `display` it will pass to the html element:

```
<svg display="..." />
```

Reply:
@heynan0 makes sense. We're passing these options to `octicon`, which ends up calling `Octicon.new`. Can we confirm that at least one of these options works:

https://github.com/primer/octicons/blob/master/lib/octicons_gem/lib/octicons/octicon.rb#L11

Reply:
Forgot to mention I had done a search for instances into how the component is used and sometimes people pass classes through `class`, sometimes through `classes`, that's why I kept the support for both.

Reply:
@heynan0 what do you think about refactoring them to always pass `class`? That would simplify things...

Reply:
@joelhawksley we talked in Slack about how this would break to the consumers of the library. You said it was OK since it's in alpha.

I also thought this:

Also I thought you had set on "classes" as the standard name? not class? 
(you replied: you meant to say `classes`)

Thinking back at `classes` vs `class` this is what I consider:
```
classes adds to the underlying component classes. e.g. Label already has "Label" class so classes adds more.

class would replace all classes in the component , overwriting the underlying ones. e.g. Label would lose "Label" if you passed class: "something-else"
```

Reply:
I changed the code so it only takes `classes`. The option `class` is no longer supported.